app/controllers/user_games_controller.rb

```diff
@@ -3,14 +3,22 @@ class UserGamesController < ApplicationController
   def create
     game = Game.find(params[:format])
     if !current_user 
-      flash[:notice] = "You must sign in to add games to Your Games"
+      flash[:notice] = "You must sign in to add games to Saved Games"
```

Comment:
Felt `Saved Games` made more sense than favorites, especially since they are not session favorites

Reply:
I agree with this change, 'Saved' makes more sense 👍